src/services/GameEngineManager.ts

```diff
@@ -28,12 +32,21 @@ export const GameEngineManager = {
       };
     }
   },
-  getState: async (lobbyId: string, userId: string): Promise<Response> => {
+  getState: async (lobbyId: string, userId: number): Promise<Response> => {
     try {
       const { data } = await gameEngineClient.get(
         `/state/${lobbyId}?userId=${userId}`,
       );
 
+      const { currentUser } = data;
+      if (!currentUser.isAlive) {
```

Comment:
Cred ca aici poate sa vine undefined .. in cazul in care `state` e `lobby` ... putem face ceva de genu:
```suggestion
      if (!data?.currentUser?.isAlive) {
```